modules/performing-an-image-signature-verification.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="performing-an-image-signature-verification_{context}"]
+= Performing an image signature verification
+
+Before you perform an image signature verification, you must first create a signature integration in {rh-rhacs-first}.
+
+When you create a signature integration, you can use the following verification methods:
+
+* Cosign public keys
+* Cosign certificates
+* Transparency log
+
+.Prerequisites
+
+. You have a Cosign public key which is encoded in Privacy Enhanced Mail (PEM) format.
++ 
+For more information about Cosign public keys, see link:https://docs.sigstore.dev/cosign/signing/overview/[Overview] (Sigstore documentation).
+
+. You have the certificate identity and issuer.
+. Optional: You have a certificate and chain which is encoded in PEM format. 
++
+For more information about Cosign certificates, see link:https://docs.sigstore.dev/cosign/verifying/verify/[Verifying Signatures] (Sigstore documentation).
+
+.Procedure
+
+. In the {product-title-short} portal, click *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Signature Integrations* section, and click *Signature*.
+. To create a new signature integration, click *New integration*.
+. Enter a name for the integration. You can save the integration only after you specify a name for the integration.
+. Optional: To add a new public key, complete the following steps:
++
+[NOTE]
+====
+** If you add a public key, you do not need to create a new certificate verification.
+** You can add one or more public keys.
+** You can save the integration only after you specify a name and value for the public key.
+====
+.. Expand *Cosign public Keys*, and then click *Add new public key*.
+.. Enter a name for the key. 
+.. Enter a value for the key which is encoded in PEM format.
+
+. Optional: To add a new certificate verification, complete the following steps:
```

Comment:
same comments as previous; this should be a separate procedure module and removed from this one